modules/system/classes/PluginManager.php

```diff
@@ -7,9 +7,9 @@
 use File;
 use Lang;
 use View;
+use Cache;
```

Comment:
Requires Cache to not be a deferred service provider because of the Laravel issue with using deferred providers in the registration process of other providers.